README.md

```diff
@@ -0,0 +1,65 @@
+# Cloudblue Connect Mulesoft Extension
+
+## Overview
+CloudBlue Connect is a supply automation platform that manages your products and services, contracts, ordering and fulfillment, usage and subscriptions.
+It supports any product, from physical goods to cloud products, as well as any channel, including your direct and indirect sales channels and internal procurement.
+Vendors can leverage CloudBlue Connect to:
+-   Reduce the total cost of ownership for homegrown technology supporting their indirect channel
+-   Standardize integrations with partners
+-   Increase efficiencies and minimize redundancies by bridging their direct and indirect sales channels
+
+Distributors can use CloudBlue Connect to:
+-   Define, manage and distribute any type of product (omni-product) through any channel (omni-channel)
+-   Transform perpetual licensed products into a subscription model
+-   Onboard new products into their portfolio quickly to build and deliver unique solutions to end customers
+
+Mulesoft Connector for Cloudblue Connect enables you to create apps that react to Cloudblue Connect events such as adding, changing, or deleting objects like purchase requests, assets, accounts.
+
+It allows an easy and fast integration with the Cloud Blue Connect Platform. Thanks to it you can automate the fulfillment of orders generated by your products.
+
+Cloudblue Connect Mulesoft Extension exposes methods for accessing Cloudblue Connect and uses the API defined in [Cloudblue Connect OpenAPI Specification](https://connect.cloudblue.com/community/api/openapi/).
```

Comment:
uses the API defined in => uses the public API defined at

Reply:
Done